src/mark.js

```diff
@@ -702,6 +719,7 @@ class Mark {
             keywords: kwArr,
             length: kwArrLen
         } = this.getSeparatedKeywords(typeof sv === "string" ? [sv] : sv);
+        const sens = opt.caseSensitive ? "" : "i";
```

Comment:
Please avoid to repeat `const` here, just write it with a `,` like you've done in `createSynonymsRegExp()`